.vortex/cli/src/Command/ConfigureCommand.php

```
@@ -105,24 +110,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $this->assertVortexProject($config);
       }
 
-      $prompt_manager = new PromptManager($config);
-      $prompt_manager->runPrompts();
+      $tui = new Engine(VortexForm::create($config), [VortexForm::HANDLER_NAMESPACE]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\VortexCli\Form\VortexForm' in method 'execute'. (undefined)

(StaticAccess)